rurusetto/wiki/forms.py

```diff
@@ -15,7 +15,7 @@ class RulesetForm(forms.ModelForm):
 
     class Meta:
         model = Ruleset
-        fields = ['name', 'description', 'icon', 'logo', 'cover_image', 'opengraph_image', 'recommend_beatmap_cover',
+        fields = ['name', 'description', 'icon', 'dark_icon', 'logo', 'cover_image', 'opengraph_image', 'recommend_beatmap_cover',
```

Comment:
```suggestion
        fields = ['name', 'description', 'icon', 'light_icon', 'logo', 'cover_image', 'opengraph_image', 'recommend_beatmap_cover',
```
Because the website's default theme is dark so it need to change.